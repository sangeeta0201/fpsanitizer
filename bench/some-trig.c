#include <stdio.h>
#include <math.h>

int main() {
  double x, y;
  x = atan(1.0) * (40002);
  printf("***x:%e\n", x);
  printf("***sin(x):%e\n", sin(x));
  printf("***cos(x):%e\n", cos(x));
  printf("***tan(x):%e\n", tan(x));
  y = tan(x) - (sin(x)/cos(x));
  printf("****y:%e\n", y);
  return 0;
}
