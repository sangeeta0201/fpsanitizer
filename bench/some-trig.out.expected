(output
  (function "main")
  (filename "some-trig.c")
  (line-num 8)
  (instr-addr 40095D)
  (avg-error 61.981567)
  (max-error 61.981567)
  (num-calls 1)
  (influences
    (
    )
  )
)
